app/styles/partials/event-card.scss

```diff
@@ -48,10 +48,6 @@
 
   .main.content {
     display: block;
-    height: 130px;
+    padding-bottom: 0px;
```

Comment:
![Screenshot from 2021-02-03 22-50-12](https://user-images.githubusercontent.com/61330148/106784519-6adb6480-6672-11eb-9d30-6167532ac5eb.png)

Not checked in button should always be at the bottom of the card.

Reply:
Thanks. Does everything else work fine?

Reply:
Your PR changes the behaviour of explore page
![Screenshot from 2021-02-03 23-03-18](https://user-images.githubusercontent.com/61330148/106786047-1cc76080-6674-11eb-973d-5ab43685ba72.png)

Reply:
Sorry to see that. Making changes. Thanks.

Reply:
Plz review it now. Thanks.